core-primitives/stf-executor/src/executor_tests.rs

```diff
@@ -117,19 +117,62 @@ pub fn propose_state_update_executes_only_one_trusted_call_given_not_enough_time
 			&vec![trusted_operation_1.clone(), trusted_operation_2.clone()],
 			&ParentchainHeaderBuilder::default().build(),
 			&shard,
-			Duration::ZERO,
+			Duration::from_nanos(50_000),
```

Comment:
Is there a way to find out an appropriate value?
On devsgx02 a value between 10_000 and 90_000 works

Reply:
I guess this one is hard.